ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -656,8 +621,9 @@ public class Manager: NSObject, UIGestureRecognizerDelegate {
 
   // MARK: - Loading custom keyboards
   /// Preloads the JS and font files required for a keyboard.
+  @available(*, deprecated, message: "Use ResourceFileManager's methods to install resources from KMPs.")
```

Comment:
This is currently utilized by the FirstVoices app; said app will get a friendly reminder-warning to be updated until we rework it to use KMPs instead.

No harm leaving it in place for now, since we fully control when and how the method goes obsolete.  This is more a warning for other 3rd party apps.